opmrun/README.md

```diff
@@ -61,6 +62,13 @@ platform, psutil, sys, re, subprocess, and tkinter as tk.
 install tkinter as tk; whereas,for Windows 10 users the tkinter package is 
 pre-installed with Python.
 
+**How to install and run:**
+1. Install Python 3.11, ensure it is in your system's PATH.
+2. Run "python setup.py build" and "python setup.py install" in the folder
```

Comment:
i really don't think we should risk the wrath of the python gods. users should pip or even better, pipx.